src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -262,6 +262,12 @@ import userInfoQuery from '@/graphql/query/userInfo.graphql';
 import loanGroupCategoriesMixin from '@/plugins/loan-group-categories';
 
 const pageQuery = gql`{
+    ui {
+      mg_donationtagline_active: uiConfigSetting(key: "mg_donationtagline_active") {
```

Comment:
make this line...
`mgDonationTaglineActive: uiConfigSetting(key: "ui.mg_donationtagline_active") {`